src/compiler/commandLineParser.ts

```diff
@@ -38,12 +38,12 @@ namespace ts {
         ["es2017.string", "lib.es2017.string.d.ts"],
         ["es2017.intl", "lib.es2017.intl.d.ts"],
         ["es2017.typedarrays", "lib.es2017.typedarrays.d.ts"],
+        ["es2018.asynciterable", "lib.es2018.asynciterable.d.ts"],
         ["es2018.intl", "lib.es2018.intl.d.ts"],
         ["es2018.promise", "lib.es2018.promise.d.ts"],
         ["es2018.regexp", "lib.es2018.regexp.d.ts"],
         ["esnext.array", "lib.esnext.array.d.ts"],
         ["esnext.symbol", "lib.esnext.symbol.d.ts"],
-        ["esnext.asynciterable", "lib.esnext.asynciterable.d.ts"],

```

Comment:
For backwards-compatibility sake we might want to have `lib.esnext.asynciterable.d.ts` that just has a `/// <reference lib="es2018.asynciterable" />`

Reply:
Actually, a simpler approach would be to have `["esnext.asynciterable", "lib.es2018.asynciterable.d.ts"],` here.

Reply:
Done!